File: Sorting/Bubble Sort/bubbleSort.cpp
#include<iostream>
using namespace std;
void Sort(int a[], int n){
    for(int i=0; i<n-1; i++){
        int flag = 0, temp;
        for(int j=0; j<n-1; j++){
            if(a[j]>a[j+1]){
                temp = a[j];
                a[j] = a[j+1];
                a[j+1] = temp;
                flag = 1;
            }
        }
        if(flag != 1) break;
    }
}
int main(){
    int n;
    cin>>n;
    int A[n];
    for(int i=0; i<n; i++){
        cin>>A[i];
    }
    Sort(A, n);
    for(int i=0; i<n; i++){
        cout<<A[i]<<" ";
    }
    cout<<endl;
    return 0;
}

File: Data Structure/Sorting/Bubble Sort/bubbleSort.java
import java.util.Scanner;

class bubbleSort{
    public static Scanner scan = new Scanner(System.in);
    static void Sort(int a[]){
        for(int i=0; i<a.length-1; i++){
            Boolean flag = false;
            int temp;   
            for(int j=0; j<a.length-1; j++){
                if(a[j]>a[j+1]){
                    temp = a[j];
                    a[j] = a[j+1];
                    a[j+1] = temp;
                    flag = true;
                }
            }
            if(flag == false) break;
        }
    }
    public static void main(String[] args) {
        int n = scan.nextInt();
        int A[] = new int[n];
        scan.nextLine();
        for(int i=0; i<n; i++){
            A[i] = scan.nextInt();
        }
        scan.close();
        Sort(A);
        for(int i=0; i<n; i++){
            System.out.print(A[i]+" ");
        }
    }
}